scripts/deploy-collector.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+
+# shellcheck disable=SC1091 # Not following.
+source "$SCRIPT_DIR"/init-env.sh
+# shellcheck disable=SC1091 # Not following.
+source "$SCRIPT_DIR"/logging.sh
+
+CHECKOUT_FOLDER=collector-deployment
+rm -rf ${CHECKOUT_FOLDER}
+
+log_info "Cloning collector"
+# git clone "$COLLECTOR_URL" -b "$COLLECTOR_TAG" "${CHECKOUT_FOLDER}" || exit 1  # TODO: uncomment this line when COLLECTOR_TAG is ready
+git clone "$COLLECTOR_URL" "${CHECKOUT_FOLDER}" || exit 1
+
+pushd "${CHECKOUT_FOLDER}" || exit 1
```

Comment:
`pushd` and `popd` is overkilling here, there is no need to preserve the stack of directories. `cd dir || exit 1; ...; cd - || exit 1;` is good enough, see the details:
 https://unix.stackexchange.com/questions/272965/pushd-popd-vs-cd-cd-in-bash-and-zsh

Reply:
We use `pushd` and `popd` in other places in the scripts as well.  I guess I'm okay with either way.